tests/test_source_to_sink.py

```diff
@@ -97,15 +106,9 @@ def check_source_to_sink(fname, source_spec, sink_spec):
         # Expect that every transaction quantity is the same amount
         expected_quantities.fill(quantities[0])
 
-        yield assert_array_equal, quantities, expected_quantities
+        assert_array_equal, quantities, expected_quantities
 
         clean_outs()
 
 
-def test_source_to_sink():
-    cases = [("source_to_sink.xml", ":agents:Source", ":agents:Sink"),
-             ("source_to_sink.py", ":cyclus.pyagents:Source", ":cyclus.pyagents:Sink"),
-             ]
-    for case in cases:
-        for x in check_source_to_sink(*case):
-            yield x
```

Comment:
Why is this test deleted?

Reply:
It's replaced with a pytest fixture above on about line 20